### PR TITLE
[AIDAPP-129]: Introduce the ability to upload a favicon image to the portal

### PR DESCRIPTION
**Pull request by ketan-canyon**
…portal

### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-129

### Technical Description

> Describe your changes in detail. This should include technical/architectural decisions and reasoning, if applicable.
> please check with hard refresh or in private window.
> Please ensure that at least one correct change type label is added to the PR. For example, if you are adding a new feature, please add the `Change Type | New Feature` label.

### Any deployment steps required?

> A deployment step could be a command that needs to be executed or an ENV key that needs to be added, for example.
>
> If yes, please describe the deployment steps required below and apply the `Deployment Steps` label to the PR.

_______________________________________________

### CODE REVIEW

File: app/Models/SettingsProperty.php [hunk 1]
```diff
@@ -68,5 +68,19 @@ public function registerMediaConversions(Media $media = null): void
...
+            ->height(512)
+            ->width(512)
+            ->acceptsMimeTypes([
+                'image/png',
+                'image/jpeg',
+                'image/ico',
+                'image/webp',
+                'image/jpg',
+                'image/svg',
+            ]);
```
Comment (Orrison) [inline app/Models/SettingsProperty.php:84]: This is not a method for conversions and as such causes an error when you try and upload. This is for the collection.

Please define the collection in this class for `portal_favicon` and apply this and other pertinent methods such as `singleFile()` there.

https://spatie.be/docs/laravel-medialibrary/v11/working-with-media-collections/defining-media-collections

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app/Models/SettingsProperty.php (hunk 1 above)
Comment (ketan-canyon) [reply app/Models/SettingsProperty.php:81]: Hi @Orrison 
Done as per requirements.
Sorry for this type of mistake. Next time i will keep this on mind.

Comment (Orrison) [reply app/Models/SettingsProperty.php:81]: No problem!

File: app/Models/SettingsProperty.php [hunk 2]
```diff
@@ -68,5 +68,22 @@ public function registerMediaConversions(Media $media = null): void
...
+        $this->addMediaCollection('portal_favicon')
+            ->singleFile()
+            ->acceptsMimeTypes([
+                'image/png',
+                'image/jpeg',
+                'image/ico',
+                'image/webp',
+                'image/jpg',
+                'image/svg',
+            ]);
```
Comment (Orrison) [inline app/Models/SettingsProperty.php:81]: This is not the right method to define Media Collections in. I believe you are supposed to do so in a `registerMediaCollections` method.

https://spatie.be/docs/laravel-medialibrary/v11/working-with-media-collections/defining-media-collections

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app/Models/SettingsProperty.php (hunk 2 above)
Comment (ketan-canyon) [reply app/Models/SettingsProperty.php:90]: Done. sorry again for silly mistakes.

Comment (Orrison) [reply app/Models/SettingsProperty.php:90]: No problem!

**Review by Orrison — APPROVED** [review]

**Pull request merged**